lib/openstudio-standards/refrigeration/create_typical_refrigeration.rb

```diff
@@ -0,0 +1,158 @@
+
+      if ref_equip_list[:cases].empty? && ref_equip_list[:walkins].empty?
+        OpenStudio.logFree(OpenStudio::Info, 'openstudio.standards.Refrigeration', 'The model space types do not typical have refrigeration cases or walkins. No refrigeration system will be added.')
+        return true
+      end
+
+      # Find the thermal zones most suited for holding the display cases
+      unless ref_equip_list[:cases].empty?
+        thermal_zone_case = OpenstudioStandards::Refrigeration.refrigeration_case_zone(model)
+        if thermal_zone_case.nil?
+          OpenStudio.logFree(OpenStudio::Error, 'openstudio.standards.Refrigeration', 'Attempted to add display cases to the model, but could find no thermal zone to put them into.')
+          return false
+        end
+      end
+
+      medium_temperature_cases = []
+      low_temperature_cases = []
+      ref_equip_list[:cases].each do |ref_case|
+        case_ = OpenstudioStandards::Refrigeration.create_case(model,
+                                                               template: template,
+                                                               case_type: ref_case[:case_type],
+                                                               case_length: ref_case[:length],
+                                                               thermal_zone: thermal_zone_case)
+        if case_.caseOperatingTemperature > -3.0
+          medium_temperature_cases << case_
+        else
+          low_temperature_cases << case_
+        end
+      end
+
+      # Find the thermal zones most suited for holding the walkins
+      unless ref_equip_list[:walkins].empty?
+        thermal_zone_walkin = OpenstudioStandards::Refrigeration.refrigeration_walkin_zone(model)
+        if thermal_zone_walkin.nil?
+          OpenStudio.logFree(OpenStudio::Error, 'openstudio.standards.Refrigeration', 'Attempted to add walkins to the model, but could find no thermal zone to put them into.')
+          return false
+        end
+      end
+
+      medium_temperature_walkins = []
+      low_temperature_walkins = []
+      ref_equip_list[:walkins].each do |walkin|
+        ref_walkin = OpenstudioStandards::Refrigeration.create_walkin(model,
+                                                                      name: walkin[:walkin_name],
+                                                                      template: template,
+                                                                      walkin_type: walkin[:walkin_type],
+                                                                      thermal_zone: thermal_zone_walkin)
+        if ref_walkin.operatingTemperature > -3.0
+          medium_temperature_walkins << ref_walkin
+        else
+          low_temperature_walkins << ref_walkin
+        end
+      end
+
+      refrigeration_space_type_area = OpenStudio.convert(model.getBuilding.floorArea, 'm^2', 'ft^2').get
+      separate_system_size_limit = 20_000.0
```

Comment:
I feel like this should be pulled out as an optional argument, defaulted to 20,000

Reply:
made this an optional argument